src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -203,16 +224,39 @@ const ModalCardDetails = ({
           'modalPayment.modalCardDetails.disclaimer.gift_card',
           sellerName
         );
+      case ModalPaymentTypes.modalPages.light_up_chinatown:
+        return t('modalPayment.modalCardDetails.disclaimer.light_up_chinatown');
       default:
         break;
     }
   };
 
+  const setDetailsText = (
+    type: string | ModalPaymentTypes.modalPages,
```

Comment:
`string | ` what is the purpose of the string in this union type - why can't it just be the enum `modalPages`

Reply:
this was  just a safety for me that i added when i was remodeling the new modal payment context, will change